IBPSA/Fluid/Interfaces/PartialTwoPortVector.mo

```diff
@@ -30,6 +30,23 @@ partial model PartialTwoPortVector "Partial component with two ports, one of whi
     each h_outflow(start=Medium.h_default, nominal=Medium.h_default))
     "Fluid connectors b (positive design flow direction is from port_a to ports_b)"
     annotation (Placement(transformation(extent={{90,-40},{110,40}})));
+
+  // Diagnostics
+   parameter Boolean show_T = false
+    "= true, if actual temperature at port is computed"
+    annotation(Dialog(tab="Advanced",group="Diagnostics"));
+
+  Medium.ThermodynamicState sta_a=
+      Medium.setState_phX(port_a.p,
+                          noEvent(actualStream(port_a.h_outflow)),
+                          noEvent(actualStream(port_a.Xi_outflow))) if
+         show_T "Medium properties in port_a";
+
+  Medium.ThermodynamicState stas_b[nPorts]=
```

Comment:
This should be `sta_b` not `stas_b` (even though it is a vector).

Reply:
@mwetter I revised the naming according to your suggestion. The naming wasn't really clear for me but with the result filtering, its a good hint that ```sta_b``` will be more user friendly.